test/admin/admin_triggers.test.js

```diff
@@ -0,0 +1,102 @@
+const StitchMongoFixture = require('../fixtures/stitch_mongo_fixture');
+import { buildAdminTestHarness, extractTestFixtureDataPoints } from '../testutil';
+import { StitchError, ErrUnauthorized } from '../../src/errors';
+
+function createSampleFunction(functions) {
+  return functions.create({
+    name: 'myFunction',
+    source: 'exports = () => {}'
+  });
+}
+
+function createSampleMongodbService(services) {
+  return services.create({
+    type: 'mongodb',
+    name: 'mdb',
+    config: {
+      uri: 'mongodb://localhost:26000'
+    }
+  });
+}
+
+function buildEventSubscriptionData(name, disabled, functionId, mongodbServiceId) {
+  return {
+    name,
+    type: 'DATABASE',
+    disabled,
+    function_id: functionId,
+    config: {
+      database: 'db',
+      collection: 'col',
+      operation_types: ['INSERT'],
+      service_id: mongodbServiceId
+    }
+  };
+}
+
+describe('Admin Triggers', () => {
+  let test = new StitchMongoFixture({ userRoles: ['GLOBAL_OWNER'] });
+  let th;
+  let eventSubscriptions;
+  let functions;
+  let services;
+  let privateAdminTriggers;
+
+  beforeAll(() => test.setup(true));
+  afterAll(() => test.teardown());
+
+  beforeEach(async() => {
+    const { apiKey, groupId, serverUrl } = extractTestFixtureDataPoints(test);
+    th = await buildAdminTestHarness(true, apiKey, groupId, serverUrl);
+    eventSubscriptions = th.app().eventSubscriptions();
+    functions = th.app().functions();
+    services = th.app().services();
+    privateAdminTriggers = th.privateAdminTriggers();
+  });
+
+  afterEach(async() => th.cleanup());
+
+  it('should return all triggers belonging to the app when list is called', async() => {
+    const fn = await createSampleFunction(functions);
+    const mongodbService = await createSampleMongodbService(services);
+
+    const eventSubscriptionName1 = 'testAdminTriggers1';
+    const eventSubscriptionName2 = 'testAdminTriggers2';
+
+    await eventSubscriptions.create(buildEventSubscriptionData(eventSubscriptionName1, false, fn._id, mongodbService._id));
+    await eventSubscriptions.create(buildEventSubscriptionData(eventSubscriptionName2, true, fn._id, mongodbService._id));
+
+    const privateAdminTriggersList = await privateAdminTriggers.list();
+    expect(privateAdminTriggersList).toHaveLength(2);
+
+    expect(privateAdminTriggersList[0].name).toBe(eventSubscriptionName1);
+    expect(privateAdminTriggersList[0].type).toBe('DATABASE');
+    expect(privateAdminTriggersList[0].state).toBe('UNOWNED');
+
+    expect(privateAdminTriggersList[1].name).toBe(eventSubscriptionName2);
+    expect(privateAdminTriggersList[1].type).toBe('DATABASE');
+    expect(privateAdminTriggersList[1].state).toBe('DISABLED');
```

Comment:
[nit] seems like you could just have this loop over `privateAdminTriggersList` and then just have these 3 assertions typed out once with indexing